### PR TITLE
[WIP] Added travis_2_actions.py

### PR DESCRIPTION
**Pull request by dherrada**
@siddacious I really should have been using some form of version control from the very beginning, but I did my best to piece it back together. Thanks for taking a look at my script.

### CODE REVIEW

File: adabot/travis_2_actions.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+
+    # Switches the badge links from linking to travis to linking to github actions
+    os.system("sed -i '/.. image::\|:target:/ s:travis-ci:github: ; s:.com/adafruit/{0}*$:.com/adafruit/{0}/actions/:; s:.svg?branch=master:/workflows/Build%20CI/badge.svg:' README.rst".format(repo))
+
+
+    os.system("git add .")
+    os.system("git diff --cached")
+    good = input("Commit?  ")
+    if good.lower() == 'y':
```
Comment (siddacious) [inline adabot/travis_2_actions.py]: This is good; I would also add a "added" or "processed" list that you add repos to when they've been done. Then you can check this list of successes before processing a repo and skip it if it's been processed already. 

You'll have to persist the list between runs by pickling and writing it to a file, or writing a json file or similar.
https://docs.python.org/3/library/pickle.html

If you also persist, load and check the lists of skipped or un-comitted repos, you'll be able to re-run the script without worrying about re-doing work.

**Comment by dherrada** [comment]
Disregard the most recent commit, that's an old version of the script that I was doing in the jankiest way possible because of permissions issues (having it print csv lines and redirecting STDOUT to the csv)

**Comment by siddacious** [comment]
@dherrada Let's revisit what to do with this once we're _done_ done with the Actions migration.

**Comment by dherrada** [comment]
@siddacious Sounds good

**Pull request closed without merging**